exchange/bitswap/bitswap.go

```diff
@@ -417,3 +417,7 @@ func (bs *Bitswap) GetWantlist() []key.Key {
 	}
 	return out
 }
+
+func (bs *Bitswap) IsOnline() bool {
```

Comment:
It is used.  I added a `IsOnline()` method to exchange to be able to tell if the exchange is online or not.